.github/workflows/create-release-pr.yml

```
@@ -0,0 +1,27 @@
+name: Create Release PR
+
+on:
+  push:
+    branches:
```

```suggestion
  push:
    types: [created]
    branches:
```

add this so that this workflow only runs when a new branch is pushed otherwise it will run on every push event